src/nnet3/nnet-example-utils.cc

```diff
@@ -1265,6 +1265,21 @@ void ExampleMerger::Finish() {
   stats_.PrintStats();
 }
 
+void ScaleFst(BaseFloat scale, fst::StdVectorFst *fst) {
```

Comment:
A function that does this already exists somewhere in fstext-utils.h.  You can remove this from the header too.